packages/compiler-cli/src/ngtsc/sourcemaps/src/source_file_loader.ts

```diff
@@ -168,6 +170,23 @@ export class SourceFileLoader {
     }
     this.currentPaths.push(path);
   }
+
+  /**
+   * Return a path to a source file that is referenced from a source-map.
+   *
+   * Some build tools replace real file paths with protocol prefixed paths - e.g. `webpack://`.
+   * We use the `protocolMap` passed to this class to convert such paths to "real" file paths.
+   * In some cases, this is not possible, since the file was actually synthesized by the build tool.
+   * But the end result is better than prefixing the sourceRoot in front of the protocol.
+   *
+   * @param sourceRoot The path to the source-map that contains this source path (or its directory).
+   * @param sourcePath The path to a source file referenced in the source-map.
+   */
+  private resolveSourcePath(sourceRoot: AbsoluteFsPath, sourcePath: string): AbsoluteFsPath {
+    const protocolMappedPath = sourcePath.replace(
+        /^([a-z]+):\/\//, (path, protocol) => this.protocolMap[protocol] || path);
```

Comment:
Should this be case insensitive?

```suggestion
        /^([a-z]+):\/\//i, (path, protocol) => this.protocolMap[protocol] || path);
```

Reply:
You are right!

> Although schemes are case-insensitive, the canonical form is lowercase and documents that specify schemes must do so with lowercase letters.